src/docdb/tree/DocDBDatabaseTreeItemBase.ts

```diff
@@ -96,16 +96,20 @@ export abstract class DocDBDatabaseTreeItemBase extends DocDBTreeItemBase<Contai
                 paths: [partitionKey]
             };
         }
-        const isFixed: boolean = !(containerDefinition.partitionKey);
-        const minThroughput = isFixed ? minThroughputFixed : minThroughputPartitioned;
-        const throughput: number = Number(await ext.ui.showInputBox({
-            value: minThroughput.toString(),
-            ignoreFocusOut: true,
-            prompt: `Initial throughput capacity, between ${minThroughput} and ${maxThroughput}`,
-            validateInput: (input: string) => validateThroughput(isFixed, input)
-        }));
-
-        const options = { offerThroughput: throughput };
+        const options: RequestOptions = {};
```

Comment:
Could you explain why this change was made?

Reply:
Options need to be defined for `throughput` when working with non-serverless database instance, and options need to be empty when working with serverless.